src/dmd/identifier.d

```diff
@@ -138,6 +138,27 @@ nothrow:
         return idPool(buf.peekSlice());
     }
 
+    /***************************************
+     * Generate deterministic named identifier based on a source location,
+     * such that the name is consistent across multiple compilations.
+     * Params:
+     *      prefix      = first part of the identifier name.
+     *      loc         = source location to use is the identifier name.
```

Comment:
`is` doesn't sound correct. Should it be `in`?

Reply:
should be `in`.